index.html

```diff
@@ -4,22 +4,27 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Simple hotel website</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css" />
+    <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-    <header>
-        <img src="images/download 1.svg" alt="Hotel logo">
-        <nav>
-            <ul>
-                <li><a href="#our hotel">Our hotel</a></li>
-                <li><a href="#rooms&rates">Rooms & rates</a></li>
-                <li><a href="#facilities">Facilities</a></li>
-                <li><a href="#contact us">Contact us</a></li>
-            </ul>
-            <button type="button"><img src="images/bi_search.svg" alt="loop icon"></button>
-        </nav>
+    <header class="page-header">
```

Comment:
this is a weird class name
class name must describe the tag styles
call it, for instance, header-layout